mlir/test/Catalyst/BufferizationTest.mlir

```diff
@@ -45,3 +45,17 @@ func.func @dbprint_str() {
 
     return
 }
+
+// -----
+
+func.func @custom_call(%arg0: tensor<3x3xf64>) -> tensor<3x3xf64> {
+    // CHECK: [[memrefArg:%.+]] = bufferization.to_memref %arg0 : memref<3x3xf64>
+    // CHECK: [[alloc:%.+]] = bufferization.alloc_tensor() {bufferization.escape = [false], memory_space = 0 : i64} : tensor<3x3xf64>
+    // CHECK: [[allocmemref:%.+]]  = bufferization.to_memref [[alloc]] : memref<3x3xf64>
```

Comment:
Why not alloc a memref directly in the bufferization?

Reply:
I am updating it :+1:

Reply:
If it's simpler all the better 💯  Didn't know about that trick :)